packages/starksheet-cairo/utils/constants.py

```diff
@@ -22,27 +22,32 @@
     if re.match(r".*(testnet|goerli)-?2$", NETWORK, flags=re.I)
     else "mainnet"
     if re.match(r".*mainnet$", NETWORK, flags=re.I)
+    else "devnetDocker"
+    if re.match(r".*devnetDocker$", NETWORK, flags=re.I)
     else "devnet"
 )
 GATEWAY_URLS = {
     "mainnet": "https://alpha-mainnet.starknet.io",
     "testnet": "https://alpha4.starknet.io",
     "testnet2": "https://alpha4-2.starknet.io",
     "devnet": "http://127.0.0.1:5050",
+    "devnetDocker": "http://starknet-devnet:5050",
```

Comment:
docker-compose use host names for container (starknet-devnet for the container running starknet-devnet), so we need a specific a conf for this host, that does not conflict with regular (local) devnets.

This would maybe deserves some refacto in this file given how the network list is repeated, but it is outside of the scope of this pull request IMO